### PR TITLE
Fix iteration

### PR DESCRIPTION
**Pull request by GwendalRaoul**

### CODE REVIEW

File: examples/example_get_remote_scratchpad_status.py [hunk 1]
```diff
@@ -30,8 +30,9 @@ def print_node_list(nodes):
     print("\nList of nodes:")
 
     id = 0
-    for node_id, node in nodes:
+    for node_id in list(nodes):
```
Comment (jy-wirepas) [inline examples/example_get_remote_scratchpad_status.py:33]: Wouldn't an iterator be more efficient here, i.e. omitting the `list()` constructor altogether?

File: wirepas_mqtt_library/wirepas_otap_helper.py [hunk 2]
```diff
@@ -214,9 +214,9 @@ def get_target_scratchpad_seq_list(self):
 
         try:
             return set([i["target_scratchpad_and_action"]['target_sequence'] for i in self._status])
-        except KeyError:
+        except (KeyError, TypeError):
             # Can happen if gateway or sink doesn't support target scratchpad
-            logging.error("Cannot determine target scratchpad sequences")
+            logging.error("Cannot determine target scratchpad sequences (are sinks and gateways supporting it?")
```
Comment (jy-wirepas) [inline wirepas_mqtt_library/wirepas_otap_helper.py:219]: Missing close paren ')' in the printed text.

The wording could be improved, e.g. "(is scratchpad valid for the sink?)" or some such.

File: examples/example_get_remote_scratchpad_status.py (hunk 1 above)
Comment (GwendalRaoul) [reply examples/example_get_remote_scratchpad_status.py:33]: No, this is exactly the reason why I'm making this change. List() will implicitly generate a copy that I need.
In fact, if you iterate on an object (dic here) that can be modified by someone else, you may end up with a _RuntimeException: dictionary changed size during iteration_

File: wirepas_mqtt_library/wirepas_otap_helper.py (hunk 2 above)
Comment (GwendalRaoul) [reply wirepas_mqtt_library/wirepas_otap_helper.py:219]: I will improve the wording. In fact the message is not clear.
The problem is not from the scratchpad itself but if you use the lib on a GW < 1.4.0 or sink <5.1.0.0

File: examples/example_get_remote_scratchpad_status.py (hunk 1 above)
Comment (jy-wirepas) [reply examples/example_get_remote_scratchpad_status.py:33]: OK

**Review by jy-wirepas — APPROVED** [review]

**Pull request merged**